src/store/test/store.ts

```diff
@@ -151,6 +151,20 @@ function stubUser(attrs: Partial<User> = {}): Partial<User> {
     firstName: 'DefaultStubFirstName',
     lastName: 'DefaultStubLastName',
     profileImage: '/default-stub-image.jpg',
+    lastSeenAt: 'default-stub-last-seen',
+    primaryZID: 'DefaultStubPrimaryZID',
+    displaySubHandle: 'DefaultStubDisplaySubHandle',
+    isOnline: false,
+    primaryWallet: null,
+    wallets: [],
+    ...attrs,
+  };
+}
+
+export function stubConversation(attrs: Partial<Channel> = {}): Partial<Channel> {
+  return {
+    id: 'channel-id',
```

Comment:
Let's update this to use the `stubCount` to ensure unique ids between stubs. Otherwise, some tests actually become invalid.